### PR TITLE
credits update

### PR DESCRIPTION
**Pull request by gwarb**

### CODE REVIEW

**Comment by jnackmclain** [comment]
You seem to have deleted can and addy and I don't see them re added based off the diff. Please verify in both PR's no one was lost.

File: _ark/ui/locale/locale_credits_keep.dta [hunk 1]
```diff
@@ -23,46 +23,76 @@
...
+(title_name
+   ""
+   "exoticri")
 (title_name
    ""
    "LocalH")
 (title_name
    ""
-   "Lyn")
+   "DarkRTA")
```
Comment (DarkRTA) [inline _ark/ui/locale/locale_credits_keep.dta]: i'd prefer to be called "Dark" if possible

**Comment by jnackmclain** [comment]
Sorry I also just notice onyxite and oktoberfest were also lost in this commit

**Comment by jnackmclain** [comment]
I'm sorry.. just notice delk was also lost. I keep re reading it

**Comment by jnackmclain** [comment]
.. and jimmy eat waffles

**Pull request merged**